scripts/entrypoint.sh

```diff
@@ -72,8 +76,10 @@ cp portmap "$HOST_CNI_BIN_PATH"
 cp aws-cni "$HOST_CNI_BIN_PATH"
 cp aws-cni-support.sh "$HOST_CNI_BIN_PATH"
 
-sed -i s/__VETHPREFIX__/"${AWS_VPC_K8S_CNI_VETHPREFIX:-"eni"}"/g 10-aws.conflist
-sed -i s/__MTU__/"${AWS_VPC_ENI_MTU:-"9001"}"/g 10-aws.conflist
+sed -i s~VETHPREFIX~"${AWS_VPC_K8S_CNI_VETHPREFIX}"~g 10-aws.conflist
+sed -i s~MTU~"${AWS_VPC_ENI_MTU}"~g 10-aws.conflist
+sed -i s~PLUGINLOGFILE~"${AWS_VPC_K8S_PLUGIN_LOG_FILE}"~g 10-aws.conflist
+sed -i s~PLUGINLOGLEVEL~"${AWS_VPC_K8s_PLUGIN_LOG_LEVEL}"~g 10-aws.conflist
```

Comment:
The above removed the `__` prefix and suffix on the variables being interpolated in the conflist file, so the sed won't work. Not sure why you changed the above from `s/__` to `s~`?

Reply:
The reason why i changed sed ~ is because of the below error, where AWS_VPC_K8S_PLUGIN_LOG_FILE=/var/log/aws-routed-eni/plugin.log 

sed -i s/__PLUGINLOGLEVEL__/"${AWS_VPC_K8S_PLUGIN_LOG_FILE}"/g test.txt
sed: -e expression #1, char 23: unknown option to `s

Reply:
The reason why i changed sed ~ is because of the below error, where AWS_VPC_K8S_PLUGIN_LOG_FILE=/var/log/aws-routed-eni/plugin.log 

sed -i s/__PLUGINLOGLEVEL__/"${AWS_VPC_K8S_PLUGIN_LOG_FILE}"/g test.txt
sed: -e expression #1, char 23: unknown option to `s